### PR TITLE
Add use case of pyhealth Trainer for the CS598 DLH PRJ CNN_ADNI

### PR DESCRIPTION
**Pull request by Chika-ZZH**
For our final project CS598, Deep Learning for Healthcare, we created a training script for Alzheimer's disease classification using a 3D convolutional neural network (AD3DCNN)on preprocessed ADNI MRI scans.
This script leverages PyHealth's Trainer framework for streamlined model
training, evaluation, and logging.

### CODE REVIEW

**Comment by jhnwu3** [comment]
Closing this PR as it lacks proper labeling. Please add appropriate labels and reopen if needed.
